lib/assessment.rb

```diff
@@ -20,20 +28,33 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_reader :age
+  attr_accessor :name, :location
```

Comment:
This should be write-only. Instead, it's readable and writeable. How would you change that?